context/src/test/groovy/io/micronaut/scheduling/ScheduledFixedRateSpec.groovy

```diff
@@ -62,24 +61,6 @@ class ScheduledFixedRateSpec extends Specification {
         }
     }
 
-    void 'test scheduled annotation with retry'() {
```

Comment:
Why was this removed?

Reply:
No test had to be split since it tested functionality from both modules and new tests were added to replace removed ones